ArcGISRuntimeSDKQt_QMLSamples/Features/ControlTimeExtentTimeSlider/ControlTimeExtentTimeSlider.qml

```diff
@@ -0,0 +1,60 @@
+// [WriteFile Name=ControlTimeExtentTimeSlider, Category=Features]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import Esri.ArcGISRuntime 100.13
+import Esri.ArcGISRuntime.Toolkit 100.13
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            id: map
+            initBasemapStyle: Enums.BasemapStyleArcGISTopographic
+
+
+            onLoadStatusChanged: {
+                            if (loadStatus !== Enums.LoadStatusLoaded)
+                                return;
+
+                            // add layer to the map
+                            operationalLayers.append(hurricanesLayer);
+                        }
```

Comment:
instead of this, simply declare the layer type as a child of the map. Layers are considered a "default property" of the map, which means that if you declare them as direct children objects, they will be appended to the operational layer list. The goal is always to make QML code as declarative as possible, so this would be a preferable workflow:

e.g.

```
Map {
    // this will get appended automatically
    ArcGISMapImagerLayer {
        url: ""
    }
}
```